schemas/node.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "NULL",
+  "title": "NODE",
+  "description": "This is the smallest unit of the whole history. It contains content: text.",
+  "type": "object",
+  "properties": {
+    "nodeId": {
+      "description": "The unique identifier for a node",
+      "type": "string",
+      "minLength": 5
+    },
+    "nodeName": {
```

Comment:
suggest: title

Reply:
But "title" is a keyword for JSON-schema )